_includes/02-image.md

```diff
@@ -0,0 +1,3 @@
+You can check out my other talks and streams at [sanket.com/talksswithdrg](https://sanket.com/talkswithdrg) and say hello on twitter [@sanket]
+(https://twitter.com/sanket)
+
```

Comment:
You can check out my other talks and streams at [sankets.com/talkswithdrg] (https://sankets.com/talkswithdrg)
and say hello on Twitter [@drgsankets] (https://twitter.com/drgsankets)